server/models/comments.js

```diff
@@ -19,7 +19,7 @@ module.exports = (sequelize, DataTypes) => {
       }
     },
     articleid: {
```

Comment:
Please specify the foreign key as 'articleId' (camel-case) in order to avoid inconsistencies and follow sequelize conventions.

Reply:
This will be fixed @TheDrizzyWay